Composer/packages/client/src/components/CreationFlow/v2/CreateOptions.tsx

```diff
@@ -242,10 +247,19 @@ export function CreateOptionsV2(props: CreateOptionsProps) {
             </ScrollablePane>
           </div>
           <div css={templateDetailContainer} data-is-scrollable="true">
+            {fetchReadMePending && <LoadingSpinner message={''} />}
```

Comment:
nit: might be a good idea if this loading has a text as well message

Reply:
Good catch, message is an optional param which defaults to 'loading...' if the message is not passed. I have changed this to not pass 'message' at all so it uses the default 'loading...' which is an acceptable message for this loader.